core/service/src/engine/centralized/endpoint.rs

```diff
@@ -419,4 +419,20 @@ impl<
 
         Ok(Response::new(response))
     }
+
+    #[tracing::instrument(skip(self, _request))]
+    async fn initiate_resharing(
+        &self,
+        _request: Request<kms_grpc::kms::v1::InitiateResharingRequest>,
+    ) -> Result<Response<kms_grpc::kms::v1::InitiateResharingResponse>, Status> {
+        unimplemented!("Resharing is not supported in Centralized KMS");
```

Comment:
I think we decided that the centralized endpoint should match the threshold API, so this probably needs to return a dummy value.